instructor/common.py

```diff
@@ -601,12 +601,19 @@ def western_blot_lysate_type(request):
     WesternBlotForm = modelform_factory(models.WesternBlot, exclude=['assignment'])
     if request.method == "POST":
         form = WesternBlotForm(request.POST, instance=wb)
+        field_names = ['has_gel_10', 'has_gel_12', 'has_gel_15']
         if form.is_valid():
-            form.save()
+            form.save(commit=False)
+            at_least_one = False
```

Comment:
The [any](https://docs.python.org/2/library/functions.html#any) function can do this too: `any((getattr(wb, field) for field in field_names))`